topics/metagenomics/tutorials/general-tutorial/tutorial.md

```diff
@@ -430,7 +481,7 @@ Let's visualize our data using Krona:
 >
 > 1. **Visualize with Krona** {% icon tool %} with the following parameters
```

Comment:
this tool could be replaced with the "taxonomy-to-krona" tool followed by the standard iuc "krona pie chart" tool (conversion tool seems to be available on usegalaxy.eu already)

Reply:
I did not manage to make it working...

Reply:
ok, I will have a look at this later, can leave it like this for now :)